packages/story-editor/src/app/media/utils/useMediaUploadQueue/index.js

```diff
@@ -318,7 +314,6 @@ function useMediaUploadQueue() {
           return;
         }
 
-        additionalData.isCropped = true;
```

Comment:
@timarney I am 99% sure this is not needed. Why was it used for?

Reply:
Ya not needed (not used) --- the initial code was copied from isMuted

```
additionalData.isMuted = true;
```